components/steps/Profile.tsx

```diff
@@ -418,13 +445,17 @@ const ProfileStep: React.FunctionComponent<Props> = props => {
               name="resume"
               id="resume"
               accept="application/pdf"
-              required
+              required={!!!profile.resume}
```

Comment:
App will let the user submit this form without selecting a resume if we already have a resume link stored in the DB. If there is no resume in the DB, they need to select a file